packages/client/components/TopBarAvatar.tsx

```diff
@@ -9,10 +9,21 @@ import {TopBarAvatar_viewer} from '~/__generated__/TopBarAvatar_viewer.graphql'
 import defaultUserAvatar from '../styles/theme/images/avatar-user.svg'
 import Avatar from './Avatar/Avatar'
 
-const SpacedAvatar = styled(Avatar)({
-  marginLeft: 8
+const AvatarWrapper = styled('button') ({
+  background: 'transparent',
+  border: 'none',
+  borderRadius: 100,
+  marginLeft: 8,
+  padding: 4,
+  ':focus': {
+    boxShadow: '0 0 0 2px #61B1EB',
+    cursor: 'pointer',
+    outline: 'none'
+  }
 })
 
+const SpacedAvatar = styled(Avatar)({})
```

Comment:
If you don’t need to style `Avatar` you can remove this and use `<Avatar />` instead of `<SpacedAvatar />` below.